src/components/ui/Container.styled.js

```diff
@@ -0,0 +1,9 @@
+import styled from 'styled-components';
+
+const Container = styled.div`
+	max-width: 800px;
```

Comment:
Same here! :) Is there any particular reason that 800px seems to be your max-width?